lib/protobuf/active_record/transformation.rb

```diff
@@ -93,20 +94,27 @@ def _protobuf_convert_fields_to_attributes(key, value)
         #     # Do some blocky stuff...
         #   end
         #
-        def attribute_from_proto(attribute, transformer = nil, &block)
-          transformer ||= block
+        #   attribute_from_proto :status, lambda { |proto| nil }, :nullify_on => :status
+        #   attribute_from_proto :status, :nullify_on => :status do |proto|
+        #     nil
+        #   end
+        #
+        def attribute_from_proto(attribute, *args, &block)
+          options = args.extract_options!
+          transformation = args.first || block
```

Comment:
This is following the previous pattern, but I think it might cleaner to simply call this `callable` here and eliminate the `else` clause below. The code will be tighter and it's one less variable to initialize.